pyproject.toml

```diff
@@ -1,3 +1,54 @@
+[project]
+
+name = "pytest"
+dynamic = [
+    "version",
+    "optional-dependencies",
+    "dependencies",
+    "scripts",
+    "requires-python"
+]
+description = "pytest: simple powerful testing with Python"
+authors = [
+    {name = "Holger Krekel"},
+]
+maintainers = [
```

Comment:
I suggest keeping this as before (only "authors") and maybe do it separately afterwards.